junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestParameterResolver.java

```diff
@@ -64,4 +75,23 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
 				ex);
 		}
 	}
+
+	// An aggregate parameter resolution requires passing all method arguments
```

Comment:
Does this comment belong here? Do we need it at all?

Reply:
Seemed a bit ambiguous to me for a potentially unknowing contributor reading the code. But i guess javadoc will do a much better job at it. Speaking of which, should i be adding documentation, or wait for the junit5 team to finalize the api?